### PR TITLE
ui: add steps to ILL and ACQ

### PR DESCRIPTION
**Pull request by ntarocco**
localhost_3000_backoffice_ill_borrowing-requests_0ff6h-cra85

localhost_3000_backoffice_acquisition_orders_ty12t-ej117

### CODE REVIEW

File: ui/src/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/BorrowingRequestPayment.js [hunk 1]
```diff
@@ -74,13 +74,15 @@ class Payment extends React.Component {
...
+    const paymentInfo = !_isEmpty(brwReq.payment);
     return (
       <>
         <Header as="h3" attached="top">
           Payment information
         </Header>
         <Segment attached className="bo-metadata-segment" id="payment-info">
-          {!_isEmpty(brwReq.payment) && <Payment brwReq={brwReq} />}
+          {paymentInfo && <Payment brwReq={brwReq} />}
+          {!paymentInfo && `No payment information`}
```
Comment (kpsherva) [inline ui/src/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/BorrowingRequestPayment.js]: how about `<InfoMessage>` here?

File: ui/src/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/BorrowingRequestSteps.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+const isBefore = (currentStatus, status) => {
+  const orderedStatuses =
+    invenioConfig.illBorrowingRequests.orderedValidStatuses;
+  return (
+    orderedStatuses.indexOf(currentStatus) < orderedStatuses.indexOf(status)
+  );
+};
+
+const Pending = ({ currentStatus, disabled }) => {
```
Comment (kpsherva) [inline ui/src/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/BorrowingRequestSteps.js]: potentially they could be a component if you think it makes sense (also the ones in acquisition)

**Review by kpsherva — APPROVED** [review]

File: ui/src/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/BorrowingRequestPayment.js (hunk 1 above)
Comment (ntarocco) [reply ui/src/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/BorrowingRequestPayment.js]: done, but I went for a normal message (no blue), it looked better IMHO.

File: ui/src/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/BorrowingRequestSteps.js (hunk 2 above)
Comment (ntarocco) [reply ui/src/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/BorrowingRequestSteps.js]: done, I still prefer to keep Order/BorrowingRequest separated even if they are similar, if ok with you.

**Comment by ntarocco** [comment]
Added a new commit to see the diff. To be squashed when merging.

**Pull request merged**